app/web/src/features/communities/events/constants.ts

```diff
@@ -1,53 +1,8 @@
 import { EventsType } from "queryKeys";
 
-export const ATTENDEES = "Attendees";
 export const details = ({ colon = false }: { colon?: boolean } = {}) =>
   `Details${colon ? ":" : ""}`;
-export const CREATE_AN_EVENT = "Create an event";
-export const CREATE_EVENT = "Create event";
-export const CREATE_EVENT_DISCLAIMER =
-  "By selecting this button, your event will be publicly displayed for the Couchers.org community to see.";
-export const DATE_REQUIRED = "Please enter a date.";
-export const EDIT_EVENT = "Edit event";
-export const END_DATE = "End date";
-export const END_TIME = "End time";
-export const END_TIME_ERROR =
-  "The event end time must be after the start time.";
-export const EVENT_DETAILS = "Event details";
-export const EVENT_DETAILS_REQUIRED = "Please fill in the event details field.";
-export const EVENT_DISCUSSION = "Event discussion";
-export const EVENT_IMAGE_INPUT_ALT = "Event image input";
-export const EVENT_LINK = "Event link";
-export const EVENTS_EMPTY_STATE = "No events at the moment.";
-export const EVENTS_LABEL = "Events";
-export const EVENTS_TITLE = "Events";
 export const getExtraAvatarCountText = (count: number) => `+${count}`;
```

Comment:
If you are up for it, feel free to take a look at the [Formatting section of the i18next](https://www.i18next.com/translation-function/formatting) and see how we might convert this to use that so we can get rid of the functions in this file altogether.

Reply:
Thanks for the hint! I am planning to use interpolation as soon as I finish the basic constants.

Reply:
ok! I'll address this later